docs/tutorials/theta_square.ipynb

```diff
@@ -204,14 +206,27 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 6,
    "metadata": {
     "ExecuteTime": {
      "end_time": "2018-06-15T12:49:48.264122Z",
      "start_time": "2018-06-15T12:49:47.511172Z"
     }
    },
-   "outputs": [],
```

Comment:
I dont think this notebook has been stripped

Reply:
fixed